mineral-ui/mineral-ui
# Update Functional Components to Fix Flow-to-Prop-Type Conversion

### Description
* `flow-to-prop-types` babel plugin does not allow props to be destructured in function declaration (Flow errors were not rendered to the console)

### Motivation and context
* Flow errors are now properly rendered

### Screenshots, videos, or demo, if appropriate

https://functional-prop-types--mineral-ui.netlify.com/

### How to test

1. Pass a prop with an unaccepted prop type to Avatar, CardActions, CardBLock, CardStatus, CardTitleMenu, Checkbox, CheckboxGroup, Choice, ChoiceGroup, Flex, FlexItem, FormFieldset, Grid, Link, MenuGroup, Radio, RadioGroup, TextInput or ThemeProvider & you should receive a console error.
2. Check any of the above component pages to ensure that correct default props are documented

### Types of changes
- Bug fix (non-breaking change which fixes an issue)

### Checklist
* [x] Rendering performance (initial load time & 60fps) and [perceived performance] [n/a](http://blog.teamtreehouse.com/perceived-performance) considered

### How does this PR make you feel?
![Dog on flamingo, flowing down a river](https://media.giphy.com/media/xTiTny6f1sogcEChRm/giphy.gif)
